config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "username": "root",
-    "password": "Mickey19",
+    "password": "kitty1",
     "database": "project2starter",
```

Comment:
Should probably update database name as well.

Reply:
Changed the database name.